tests/crypto/Kconfig

```diff
@@ -0,0 +1,10 @@
+config TV_ASSERT_USER_OVERRIDE
+	bool "Override test vector asserts"
+    default n
+	help
+	  Enable overriding test vector assert defines in common_test.h.
+	  If True the user should provide tv_assert_user_override.h in Zephyr's include path,
+	  e.g. by adding zephyr_include_directories(project PRIVATE my_folder) to a project's CMakeLists.txt.
+	  The override header may now #define the various TEST_* macros in common_test.h.
+
+source "Kconfig.zephyr"
```

Comment:
Newline at end of file to keep github happy?

Reply:
Thanks, amended in a4143c51